### PR TITLE
docs: update docs for integration test

### PR DESCRIPTION
**Pull request by Xynnn007**
Update the docs for integration test, which is useful for readers using the test

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
-* Pull manifefst of `"docker.io/arronwang/busybox_kbs_encrypted"` without verification of signature.
+* Pull manifest of the image without verification of signature.
 * Pull layers of the mentioned image.
 * Ocicrypt-rs will ask the Attestation-Agent to decrypt the Layer Encryption Key (LEK for short), which is 
 encrypted using Key Encryption Key (KEK for short). KEK is hard-coded in sample-kbc.
 * Ocicrypt-rs decrypt the layers using LEK. Finish the image pulling.
 
-The way to genetate KBS encrypted image please refer to
-[the guide](https://github.com/confidential-containers/image-rs/blob/main/test_data/generate_test_data.md)
+Different KBC uses different protocol format, so different KBS is needed to
```
Comment (fitzthum) [inline tests/README.md:27]: `Different KBCs use different protocol formats, so different KBSs are needed to...`

File: tests/README.md [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
+encrypt the images. The way to genetate KBS encrypted image please refer to the following link:
```
Comment (fitzthum) [inline tests/README.md:28]: `To generate encrypted image please...`

File: tests/README.md [hunk 3]
```diff
@@ -1,27 +1,34 @@
...
+
 ## Image Decryption
 
 Implemented in `image_decryption.rs`.
 
 Image decryption will cover both Sample KBC and Offline-fs-kbc:
-* Sample KBC uses `"docker.io/arronwang/busybox_kbs_encrypted"`
-* Offline-fs-kbc uses `docker.io/xynnn007/busybox:encrypted`
+* `Sample Kbc` uses `"docker.io/arronwang/busybox_kbs_encrypted"`
+* `Offline-fs-kbc` uses `docker.io/xynnn007/busybox:encrypted`
```
Comment (fitzthum) [inline tests/README.md:17]: One of the containers is in quotes but the other isn't?

**Review by fitzthum — APPROVED** [review]
See comments for a few minor tweaks. Looks fine overall.

**Comment by Xynnn007** [comment]
Thanks for your review, @fitzthum ! I've changed following your suggestions. Please take some time to review again

**Review by jialez0 — APPROVED** [review]
LGTM.

**Pull request merged**